documentation/API_documentation/Network Slice Booking API Design.md

```diff
@@ -0,0 +1,77 @@
+# Network Slice Booking API Design
+
+## **Use Cases**
+Scenario can be  B2B, B2C, B2B2C
+
+1. Scenario 1 - B2B
+
+   Sell Slice Directly
+
+Customer: toB Enterprise Customer
+
+Seller: Operator 
+
+Mode: Operator provide slice services by NaaS API to enterprise customers directly. 
+
+Example Use Cases: In the past/current, most of  current slice customer need to order slice from operators directly through operator customer managemer. This takes time from 2 weeks to 2 month. Those kind of customers may renew, reopen, modify, close the slice service using NaaS Slice APIs. 
+
+
+
+
+2. Scenario 2 - B2C
+
+OTT act as a Platform to Sell Slice
+
+Customer: Individuals  / Studios / Influencers
+
+Selling Platform: OTT
+
+Seller: Operator
+
+Mode: OTT only act as a platform who help provide slice to customer. 
+
+Step1: OTT code and embed the API in the APP, provide the channel for individuals (to C) to purchase the slice. 
+
+Step2: Individual purchase the slice from the OTT APP, pay to operators directly. 
+
+Step3: Provider provide the slice to the individuals (to C) directly. 
+
+
+
+3. Scenario3 - B2B2C
+
+   OTT buy the slice, and resell the slice. 
```

Comment:
The OTT refers here to an aggregator, who then resells the service to Enterprises or Consumers, correct?

Reply:
OTT is enterprises. Resell to OTT toC customer. B to B to C. Operators to OTT to C End Customers.